### ISSUE TITLE
Allow mentors to mark solution as "requires action"

### ISSUE THREAD

**Issue by SleeplessByte** [exercism/exercism#4679]
Currently once a solution is approved, even if a student asks a question, this solution will never "require action".

- new submissions should mark mentoring as stale
- mentors should be able to track a solution, so if they don't have time to respond to the notification, it's not lost forever

**Comment by iHiD** [comment]
Good idea. In the interim, you can see notifications in your "all notifications"

**Comment by iHiD** [comment]
If you want to open another issue for "Mark as unread" on notifications, I'll do that too.

**Comment by SleeplessByte** [comment]
> Good idea. In the interim, you can see notifications in your "all notifications"

yeah, the problem is: I have 30+ pages now :P

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂
